cmd/gapir/cc/main.cpp

```diff
@@ -821,7 +926,8 @@ static int startServer(core::CrashHandler* crashHandler, Options opts) {
 
 // Main function for PC
 int main(int argc, const char* argv[]) {
-  Options opts = Options::Parse(argc, argv);
+  Options opts;
```

Comment:
Any particular reason to switch this to passing by ref, and filling in?

Passing by ref violates the style guide as well. If you go down this route, you should pass by pointer. Style guide says you should only ever pass things by const-ref (which obviously won't work in this case)

Reply:
```
Options opts = Options::Parse(argc, argv);
```
Should have RVO applied, so you shouldn't end up paying any cost for the "copy".

Reply:
The pass by reference is not for "performance". It allows me to fill in some default values on android (that are gleaned from the apk) before later parsing the flags and applying the appropriate state changes on top.

I agree that copy elision should kick in here. And anyway the structure is so small and the operation once off, so optimizing it at the expense of clarity makes no sense.

Happily, I prefer passing things by pointer than non-cost reference. I used a reference at @hevrard  proposal. Would you prefer a pointer here?

Reply:
Understood!

We try our best to obey the style guide, so it would be better to make it a pointer.